exchanges/orderbook/orderbook.go

```diff
@@ -112,13 +111,15 @@ func (s *Service) DeployDepth(exchange string, p currency.Pair, a asset.Item) (*
 		s.books[strings.ToLower(exchange)] = m1
 	}
 	book, ok := m1.m[mapKey]
-	if !ok {
-		book = NewDepth(m1.ID)
-		book.exchange = exchange
-		book.pair = p
-		book.asset = a
-		m1.m[mapKey] = book
-	}
+	if ok {
+		// Maybe in future we should return an error here and be more strict.
```

Comment:
As in, check the order book for errors and return them?

Reply:
No I was more meaning to disallow the book to be deployed twice as it should only be used in the context of websocket but it broke a bunch of tests across the code base so I opted to [revert that change](https://github.com/thrasher-corp/gocryptotrader/pull/1682/commits/ce7ba495e3d45ac4c526e563c42a56b59b5cb462).